src/ElFinder.php

```diff
@@ -356,7 +356,7 @@ public function __construct($opts) {
                             if (is_string($handler) && strpos($handler, '.')) {
                                 list($_domain, $_name, $_method) = array_pad(explode('.', $handler), 3, '');
                                 if (strcasecmp($_domain, 'plugin') === 0) {
-                                    if ($plugin = $this->getPluginInstance($_name, isset($opts['plugin'][$_name])? $opts['plugin'][$_name] : array())
+                                    if ($plugin = $this->getPluginInstance($_name, isset($opts['plugins'][$_name])? $opts['plugins'][$_name] : array())
```

Comment:
Not sure about this change, comparing with this one https://github.com/Studio-42/elFinder/blob/master/php/elFinder.class.php#L424